app/src/main/jni/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+add_library(capture
+            SHARED
+            capture.cpp )
+
+find_library(log
+             log )
+
+target_link_libraries(  capture
+                        log )
```

again: formatting.
findLibrary with log log?
```suggestion
find_library(
        log
)

target_link_libraries(
        capture
        log
)
```
ALWAYS DO SELF REVIEW BEFORE PUSHING THE CHANGE

---

find_library command require at least two parameters